compositor_pipeline/src/pipeline/rtp/rtp_input/rtcp_sync.rs

```diff
@@ -0,0 +1,209 @@
+use std::{
+    sync::{Arc, RwLock},
+    time::{Duration, Instant},
+};
+
+use tracing::{debug, warn};
+
+use crate::pipeline::rtp::rtp_input::rollover_state::RolloverState;
+
+const POW_2_32: f64 = (1i64 << 32) as f64;
+
+#[derive(Debug)]
+/// State that should be shared between different RTP tracks to use for synchronization.
+pub(crate) struct RtpNtpSyncPoint {
+    sync_point: Instant,
```

Comment:
Technically, sync_point is not functionally  necessary in this structs, but I think it fits here